src/scrabble.js

```diff
@@ -1,5 +1,31 @@
-function scrabble() {
-  // write code here
+function scrabble(scrabbleword) {
+
+  let word = scrabbleword
+  
+  if (word === null) {
+    return 0
+  }
+  if (word === '' || word === " \t\n") {
+    return 0
+  }
+  word = word.toUpperCase();
+  const lettervalues = {
+  'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
```

Comment:
I like the layout of this since its really easy to understand and easy on the eye's